main.go

```diff
@@ -2853,6 +2888,7 @@ Currently, only the --renew option is supported, and it is required:
 			continue
 		}
 
+		os.Unsetenv("VAULT_SKIP_VERIFY")
```

Comment:
Does this belong in this PR?

Reply:
Sadly yes, as the `VAULT_SKIP_VERIFY` variable is conditionally set and should originate only from configuration. This combination made the reset necessary.

Reply:
I believe there was a reason we don't unset it.  I'll have to check with other contributors to see if they can recall what it was.